apps/notifications/package.json

```diff
@@ -22,7 +22,7 @@
 		"start": "yarn run clean && yarn run build:notifications && yarn run dev-server",
 		"dev": "yarn run calypso-apps-builder --localPath dist --remotePath /home/wpcom/public_html/widgets.wp.com/notifications",
 		"build": "NODE_ENV=production yarn dev",
-		"build-ci": "yarn build && ./bin/prepArtifactForCI.sh"
```

Comment:
Prep artifact actually requires the "prev build" to be downloaded already, so it should be in the normalization script